versioning/invoke_embedme.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# Python version 3.4 or higher is required to run this script.
+
+
+
+import argparse
+import os
+import sys
+import urllib.request
+
+# NPX command.
+npx_command = 'npx embedme'
+
+def delete_from_file(the_file):
```

Comment:
Maybe `delete_file_if_exists` is a better name?

Reply:
I like delete_from_file better